README.md

```diff
@@ -74,6 +82,27 @@ accept either the name or the ID of the branch, and you can use both (_i.e._,
 the `compare_branch` can use the branch name while the `base_branch` uses the
 branch ID or vice-versa).
 
+For the action to be able to create PR comments you must add the correct
+permissions to the job. To do this add the following permissions to your job:
+
+```yml
+jobs:
+  your_job:
+    permisions:
+      pull-request: write
+      ...other permissions needed for the rest of the job
+    steps:
+      - uses: neondatabase/schema-diff-action@v1
+      ...
+```
+
```

Comment:
When setting permissions, make sure to account for all actions your workflow performs to avoid missing any required permissions. For example, if your repository is private, you must also grant read or write access to the repository with `contents: write`.

For a complete list of permissions, refer to the GitHub documentation on [Defining access for the GITHUB_TOKEN permissions](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#defining-access-for-the-github_token-permissions).